nf_core/lint.py

```diff
@@ -447,6 +447,34 @@ def check_conda_env_yaml(self, api_timeout=10):
                                 break
                     else:
                         self.failed.append((8, "Could not find Conda dependency using the Anaconda API: {}".format(dep)))
+            if isinstance(dep, dict):
+                # Check if dependency is from packages on PyPi
+                if dep.get('pip'):
+                    for pip_dep in dep.get('pip', []):
```

Comment:
..though you could probably _just_ do this and skip the above `if` statement.. 😉

Reply:
true